custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -238,6 +243,17 @@ func (t *Translator) GetMetricKind(metricName string) (string, error) {
 	return response.MetricKind, nil
 }
 
+// GetExternalMetricProject If the metric has "resource.labels.project_id" as a selector, then use a different project
+func (t *Translator) GetExternalMetricProject(metricSelector labels.Selector) (string, error) {
+	requirements, _ := metricSelector.Requirements()
+	for _, req := range requirements {
+		if req.Key() == "resource.labels.project_id" {
```

Comment:
Please make sure that req.Operator() is selection.Equals (or selection.DoubleEquals), otherwise an error should be returned.